### PR TITLE
Simplifying SSE docs and referring to KES docs for integrations

### PR DESCRIPTION
**Pull request by djwfyi**
This PR simplifies the management of KMS integrations by removing the detailed documentation and linking out to the KES docs site instead.

This completes work started on the KES docs side in https://github.com/minio/kes-docs/pull/48.

Staged:
http://192.241.195.202:9000/staging/sse-kes/linux/operations/server-side-encryption.html

Note: The steps are conditioned for different OS bases, but only Linux is staged.

### CODE REVIEW

File: source/administration/server-side-encryption.rst [hunk 1]
```diff
@@ -51,10 +51,12 @@ supports the following encryption strategies:
       Deployment, see :ref:`minio-encryption-sse-kms-quickstart`. For
       production MinIO deployments, use one of the following guides:
 
-      - :ref:`AWS SecretsManager <minio-sse-aws>`
-      - :ref:`Google Cloud SecretManager <minio-sse-gcp>`
-      - :ref:`Azure Key Vault <minio-sse-azure>`
-      - :ref:`Hashicorp KeyVault <minio-sse-vault>`
+      - :kes-docs:`Azure Key Vault <integrations/azure-keyvault/>`
+      - :kes-docs:`AWS SecretsManager <integrations/aws-secrets-manager/>`
```
Comment (feorlen) [inline source/administration/server-side-encryption.rst:55]: Should this be `Secrets Manager` or `Secrets Manager`? The linked KES page says `Secrets Manager`

File: source/administration/server-side-encryption.rst [hunk 2]
```diff
@@ -51,10 +51,12 @@ supports the following encryption strategies:
...
+      - :kes-docs:`Fortanix SDKMS <integrations/fortanix-sdkms/>`
+      - :kes-docs:`Google Cloud SecretManager <integrations/google-cloud-secret-manager/>`
```
Comment (feorlen) [inline source/administration/server-side-encryption.rst:57]: Similar `SecretManager` vs `Secret Manager`

File: source/administration/server-side-encryption.rst [hunk 3]
```diff
@@ -51,10 +51,12 @@ supports the following encryption strategies:
...
+      - :kes-docs:`Hashicorp KeyVault <integrations/hashicorp-vault-keystore/>`
```
Comment (feorlen) [inline source/administration/server-side-encryption.rst:58]:
```suggestion
      - :kes-docs:`Hashicorp Vault <integrations/hashicorp-vault-keystore/>`
```

Comment (feorlen) [reply source/administration/server-side-encryption.rst:58]: The product name typos are mainly fix once, copy elsewhere so I didn't note each one.

File: source/administration/server-side-encryption.rst (hunk 1 above)
Comment (djwfyi) [reply source/administration/server-side-encryption.rst:55]: Requires the space per Amazon. Fixed throughout.
Thanks for the catch!

File: source/administration/server-side-encryption.rst (hunk 2 above)
Comment (djwfyi) [reply source/administration/server-side-encryption.rst:57]: Requires the space per Google, fixed throughout!

File: source/administration/server-side-encryption.rst (hunk 3 above)
Comment (djwfyi) [reply source/administration/server-side-encryption.rst:58]: Fixed throughout, thanks for the catch.

File: source/operations/server-side-encryption.rst [hunk 4]
```diff
@@ -83,11 +88,73 @@ supports the following encryption strategies:
...
+---------------------------
 
-   /operations/server-side-encryption/configure-minio-kes-hashicorp
-   /operations/server-side-encryption/configure-minio-kes-aws
-   /operations/server-side-encryption/configure-minio-kes-gcp
-   /operations/server-side-encryption/configure-minio-kes-azure
\ No newline at end of file
+.. cond:: linux
+
+   This procedure provides guidance for deploying MinIO configured to use KES and enable :ref:`Server Side Encryption <minio-sse-data-encryption>`.
```
Comment (ravindk89) [inline source/operations/server-side-encryption.rst:96]: We're saying "this procedure" but there are no steps below. We should at least have the MinIO steps right?

**Comment by djwfyi** [comment]
Redoing this effort in #1136

**Comment by djwfyi** [comment]
Closing, as done in #1136

**Pull request closed without merging**